lib/ledger_sync/serialization/type/deserializer_references_many_type.rb

```diff
@@ -11,7 +11,7 @@ def cast_value(args = {})
           value                  = args.fetch(:value)
           resource               = args.fetch(:resource)
 
-          return if value.nil?
+          return [] if value.nil?
```

Comment:
In this proposition (thanks @jozefvaclavik for your time and your research), we're being more precise/meticulous.

We're working directly on the references_many file.

Nonetheless, we also touched the `value_mixin` file (see below) which could lead to edge cases. I cannot guarantee here. I did several tests with several values and types. I put some breaking points in the code execution trying to understand and check. But still.